frontend/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState} from 'react';
+import ToDoList from './Components/ToDoList'
+import CreateItemForm from './Components/CreateItemForm'
+import {Container,Grid} from '@mui/material';
+import './App.css';
+export default function App() {
+  const [todoItems, setTodoItems] = useState({});
+  const [todoItemsOrderedList,setTodoItemsOrderedList] = useState([]);
+
+  const createToDoItem = (itemName)=>{
+    const newToDo = {
+      itemName : itemName,
+      isCompleted : false,
+      position : 0, 
```

Comment:
rather than have the block of conditional logic below, could set the position here:

```suggestion
      position : todoItems.length,  // assuming todoItems is an array
```

Reply:
Might also be good to add a unique `id` field, spoilers for the upcoming database part, but an ID would help keep things more easily organized